docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -21,118 +23,96 @@ and a packaged todo application with a custom suite of tests.
 [[synthetics-quickstart-step-two]]
-== Step 2: Update `heartbeat.docker.yml`
+== Step 2: Create a heartbeat.yml configuration file
 
 There are two ways to configure a synthetic test.
 Let's take a quick look at each.
 
 Run an inline test::
 
-If you're running an inline, browser based test, you can use the traditional Heartbeat flow to completely
-configure your synthetic testing directly in `heartbeat.docker.yml`.
-An example is provided:
-+
+If you're running an inline browser based test, you can use the traditional Heartbeat flow to completely
+configure your synthetic testing directly in a file named `heartbeat.yml` as shown below. 
+
+You'll need to setup your elasticsearch credentials for either an https://www.elastic.co/guide/en/beats/heartbeat/current/configure-cloud-id.html[Elastic Cloud ID] or another https://www.elastic.co/guide/en/beats/heartbeat/current/elasticsearch-output.html[Elasticsearch Cluster] in
+the `heartbeat.yml` file. The example below demonstrates using https://cloud.elastic.co[Elastic Cloud] credentials.
+
 [source,yml]
 ----
 heartbeat.monitors:
 - type: browser
   id: my-monitor <1>
   name: My Monitor
   schedule: "@every 1m"
-  script: |- <2>
-    step("load homepage", async () => {
-        await page.goto('https://www.elastic.co');
-    });
-    step("hover over products menu", async () => {
-        await page.hover('css=[data-nav-item=products]');
-    });
+  source:
+    inline:
+      script: |- <2>
+        step("load homepage", async () => {
+            await page.goto('https://www.elastic.co');
+        });
+        step("hover over products menu", async () => {
+            await page.hover('css=[data-nav-item=products]');
+        });
 ----
 <1> Each `monitor` gets its own ID in the {uptime-app} and, therefore its own schedule entry.
 This allows tests to be run in parallel and analyzed separately.
 <2> In this example, a synthetic test is defined inline. This is a two-step script that first loads
 a homepage and then hovers over a product menu. See <<synthetics-syntax>> for more information.
 
+
 Run a test suite::
 
 If you'd like to run multiple tests, you'd probably be better off creating a library of tests defined
 outside of `heartbeat.docker.yml`.
 This, for example, allows your tests to live with your application or in a separate Git repository.
-+
+
 [source,yml]
 ----
-heartbeat.synthetic_suites: <1>
 - name: Todos
-  path: "/opt/examples/todos" <2>
+  id: todos
+  type: browser
   schedule: "@every 1m"
+  source:
+    local:
+      path: "/opt/examples/todos" <1>
 ----
-<1> Specify the name, path, and schedule of your test suite.
-<2> In this example, our library of synthetic tests live in the `/examples/todos` directory of `elastic/synthetics`.
+<1> In this example, our library of synthetic tests come from https://github.com/elastic/synthetics/tree/master/examples/todos[our todos example]. The path shown here corresponds to the path heartbeat will look at inside the docker container.
```

Comment:
Might be worth calling out how this relates to the configuration, something along the lines:

> ...as defined with the `--volume` parameter to `docker run` as shown below

Reply:
Improved in the latest changes